app/uk/gov/hmrc/merchandiseinbaggagefrontend/forms/SearchGoodsCountryFormProvider.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggagefrontend.forms
+
+import javax.inject.Inject
+import play.api.data.Form
+import uk.gov.hmrc.merchandiseinbaggagefrontend.forms.mappings.Mappings
+
+class SearchGoodsCountryFormProvider @Inject() extends Mappings {
```

Comment:
A question not an issue. Why FormProviders and not just Forms?

Reply:
No real reason, just following a pattern I've used in previous projects, happy for them to be refactored